crates/miden-lib/asm/miden/contracts/auth/multisig.masm

```diff
@@ -0,0 +1,282 @@
+use.miden::account
```

Comment:
This file seems to be the same as `auth/multisig.masm`

But anyway after the merge (and as per the comment above) the contents should probably go directly into `account_components` directory